src/components/BreakoutRoomsEditor/SelectableParticipant.vue

```diff
@@ -113,6 +103,28 @@ export default {
 		&:has(:active) {
 			background-color: var(--color-primary-light-hover);
 		}
+
+		&::after {
+			position: absolute;
+			top: 45%;
+			right: calc(4 * var(--default-grid-baseline));
+			content: "";
+			/* short arm */
+			width: 5px;
+			border-bottom: var(--border-width-input-focused) solid var(--color-main-text);
+			/* long arm */
+			height: 11px;
+			border-right: var(--border-width-input-focused) solid var(--color-main-text);
+			/* rotate to make it a check icon */
+			transform: translateY(-50%) rotate(45deg);
+		}
```

Comment:
By CSS I meant, just show/hide the icon using `:checked` and `display: none` CSS, not fully implement an icon 🙈

Reply:
🙈